### PR TITLE
Erling/2.1/search for patient

### PR DESCRIPTION
**Pull request by erlinvan**

### CODE REVIEW

File: src/dataexplorer/dataexplorer-application/src/components/PatientInfo/PatientInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import useFetch from 'utils/hooks/useFetch';
+import { IPatient } from 'types/IPatient';
+
+const PatientInfo = ({ patientID }: any) => {
+    const { response } = useFetch<IPatient>('fhir/Patient/' + patientID);
+
+    console.log(response);
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/PatientInfo/PatientInfo.tsx:8]: I'm not a big fan of console logs in production code. What do you think?

File: src/dataexplorer/dataexplorer-application/src/components/SearchBox/SearchBox.tsx [hunk 2]
```diff
@@ -12,52 +12,76 @@ const SearchBox = () => {
...
-            isHuman: true,
+            id: '1',
+            birthDate: '2015-01-02',
+            gender: 'male',
+            name: [
+                {
+                    family: 'Er',
+                    given: ['Erling'],
+                },
+            ],
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/SearchBox/SearchBox.tsx:23]: Remove unused testdata?

File: src/dataexplorer/dataexplorer-application/src/components/PatientInfo/PatientInfo.tsx (hunk 1 above)
Comment (EnvyPhD) [inline src/dataexplorer/dataexplorer-application/src/components/PatientInfo/PatientInfo.tsx:8]: Remove console.log + whitespace?

Comment (erlinvan) [reply src/dataexplorer/dataexplorer-application/src/components/PatientInfo/PatientInfo.tsx]: fixed

Comment (erlinvan) [reply src/dataexplorer/dataexplorer-application/src/components/PatientInfo/PatientInfo.tsx]: I agree

**Pull request merged**